src/test/test_runner.py

```diff
@@ -125,6 +126,7 @@ def test_prepare_environment_with_git(self):
                 "GIT_COMMIT": conf.git_commit,
                 "GIT_BRANCH": conf.git_branch,
                 "GIT_SHA_SHORT": conf.git_sha_short,
+                "GIT_REMOTE_ORIGIN_URL": scm.get_remote_url(repo),
```

Comment:
the same comment applies here, that is, we would like to not invoke `popper.scm` directly but instead read the value from the config object.